mirage/config.js

```diff
@@ -589,6 +589,62 @@ export default function () {
   this.get('/v3/enterprise_license', function (schema, request) {
     return new Response(404, {}, {});
   });
+
+  this.get('/insights/metrics', function (schema, {queryParams}) {
```

Comment:
I would suggest to keep spaces between parameters in destructuring statements, e.g. `{ queryParams }` - it makes them more distinguishable.

Reply:
Change made :)